test/omit.spec.js

```diff
@@ -0,0 +1,24 @@
+import { omit, fromPayload } from '../src'
+
+const reducer = omit(fromPayload('ids'))
+
+describe('omit', () => {
+  test('omits arrays', () => {
+    const output = reducer([0, 1, 2, 3, 4], { payload: { ids: [0, 2, 4] } })
+    expect(output).toEqual([1, 3])
```

Comment:
Just one detail - From this test it's not clear whether omit is based on values or keys. It's not until you watch the next test (or look for ramda documentation on `omit`) when it becomes obvious.

I'd give the initial array other values so it loses the ambiguity

Reply:
That's a really good point, I'll change the test, thanks!. TBH those tests suck, I just wrote the very basic tests to make sure that this was kinda working... My idea was to improve the tests during this week.

Also, I was not sure whether to make `omit` work with `Array`s or not. For instance the `omit` of ramda is just for objects... But I thought that it wouldn't hurt.